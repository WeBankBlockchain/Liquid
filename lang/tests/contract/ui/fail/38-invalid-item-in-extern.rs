use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    extern "liquid" {
        fn bar(&self);

        #[link_name = "environ"]
        static libc_environ: *const *const std::os::raw::c_char;
    }
}

fn main() {}
