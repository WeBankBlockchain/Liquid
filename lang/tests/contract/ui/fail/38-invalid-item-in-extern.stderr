error: only function declaration is allowed for the `extern` block
 --> $DIR/38-invalid-item-in-extern.rs:8:9
  |
8 | /         #[link_name = "environ"]
9 | |         static libc_environ: *const *const std::os::raw::c_char;
  | |________________________________________________________________^
